exp-models/addon/authenticators/jam-osf-jwt.js

```diff
@@ -23,32 +20,27 @@ export default Base.extend({
                     access_token: accessToken
                 }
             }})
-        }).then(function(res) {
+        }).error(function(/*xhr, status, error*/) {
```

Comment:
I may be going senile, but where is the `.error` method coming from?

JQuery [deferreds](https://api.jquery.com/category/deferred-object/), for example, use `.done` and `.fail`, while the A+ promises [specification](https://promisesaplus.com/) mainly seems to rely on the single `.then(done, fail)` method.